scripts/pages/pgWelcome.ts

```diff
@@ -16,7 +19,10 @@ export default class PgWelcome extends withDismissAndBackButton(PgWelcomeDesign)
     }
     onShow() {
         super.onShow();
-        this.initDismissButton(this.router);
+        this.initDismissButton(this.router, {
+            color: themeService.getStyle('.sf-headerBar.itemColor'),
+            image: Image.createFromFile(`images://${image}`)
```

Comment:
Does all dismiss icons changed? If so we should change default one

Reply:
OK.